kmk/boards/handwire_ghirkin.py

```diff
@@ -0,0 +1,12 @@
+import board
+
+from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.matrix import DiodeOrientation
+
+
+class KMKKeyboard(_KMKKeyboard):
+    col_pins = (board.SDA, board.A0, board.A1, board.A3, board.A2, board.A4)
+    row_pins = (board.MOSI, board.MISO, board.RX, board.TX, board.D2)
```

Comment:
For hand-wire boards, `col_pins` and `row_pins` should live in the user keymap instead of a global definition, yeah?

Heck, this whole file probably shouldn't exist if we're roughly keeping with historical trend (the `klarank` should probably also not be in `boards` but whatever). The `user_keymap` should likely be the only file for handwires.

Reply:
I seemed to have some issues putting this in a keymap file, but it could have been the constant corruption on my microcontroller. I separated the lines based on another board I found, so oops.